pageserver/src/tenant/remote_timeline_client.rs

```diff
@@ -993,6 +1005,10 @@ impl RemoteTimelineClient {
                 return;
             }
 
+            // on cancellation these will all fail with an error... which sadly is deep in the
+            // anyhow, so there is a chance of dumping unnecessary stacktraces.
+            //
+            // after that, we'll loop around to above, and shutdown everything.
```

Comment:
Can't we use this to get to filter out the cancellations as root cause? https://docs.rs/anyhow/latest/anyhow/struct.Error.html#method.root_cause

Reply:
No need. And that would still be a string match.

Reply:
Aah in case you were wondering for the logging stacktraces, yeah for that a named type and root_cause would work. But this will work correctly otherwise.

Reply:
> Aah in case you were wondering for the logging stacktraces, yeah for that a named type and root_cause would work. 
Yeah, that was my train of thought.

> But this will work correctly otherwise.

Yep, just log spam.